tests/test_events.py

```diff
@@ -10,8 +10,18 @@ def test_post_events(
     auth_keys = [{"kid": "kid1", "secret": "secret1"},
                  {"kid": "kid2", "secret": "secret2"}]
     client = client_factory(auth_keys)
+    body = [{
+        "course_id": 0,
```

Comment:
For auth related tests, I would just pass `[]` since that should work and that way you're not trying to test two things at once (e.g. specific events and auth).